apps/system/js/hierarchy_manager.js

```diff
@@ -58,6 +58,7 @@
      * @type {Array}
      */
     PRIORITIES: [
+      'OverlayWindowManager',
```

Comment:
This is a good point.
Given that the only use case that we have so far for the GOWM is BuddyUp I would say that we shouldn't even allow placing the GOWM while the screen reader is active and we should hide it if it is already active. For BuddyUp we need the GOWM to draw the coordinates where the user should click. But screen reader is supposed to be for users that can't see the screen, so drawing on it makes no sense. Instead, BuddyUp should try a different approach to guide the user if the screen reader is active.
@rik what do you think?